pkgs/applications/video/mpv/scripts/sponsorblock.nix

```
@@ -0,0 +1,52 @@
+{ stdenv, fetchFromGitHub, fetchpatch, python3 }:
+
+stdenv.mkDerivation rec {
+  pname = "mpv_sponsorblock";
+  version = "f71e49e"; # 2020-07-05
```

```suggestion
  version = "unstable-2020-07-05";
```

See: https://nixos.org/nixpkgs/manual/#sec-package-naming

---

Amended.